components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -818,6 +834,122 @@ public static String getUserConsentURL(OAuth2Parameters params, String loggedInU
         return consentPage;
     }
 
+    private static String getConsentPageURLWithFilteredParams(SessionDataCacheEntry entry, String consentPage) {
+
+        if (isAuthEndpointRedirectParamsConfigAvailable()) {
```

Comment:
Change the method name

Reply:
Resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2020/commits/ba3d3ded0c372f6c6846cbb176d101e83a71c105